### PR TITLE
Added support for pass-otp

### PR DESCRIPTION
**Pull request by msmol**
Convention: otp passwords must have the string "totp-" either in
the filename or the file path.

If the convention is followed, "pass otp" is called instead of
"pass", so that the totp token is copied to clipboard rather
than the raw otpauth:// URI.

### CODE REVIEW

File: pass.cpp
Comment (akermu) [inline pass.cpp]: Can you split the the path into its parts and check if any part starts with `totp-`?

**Comment by akermu** [comment]
Thanks for the contribution!

**Comment by msmol** [comment]
Hi, sorry, made a quick change!

**Comment by msmol** [comment]
One more thing: I just realized I should update the readme. I'll do that and it'll be ready to merge (if you want!) :-)

File: pass.cpp
Comment (msmol) [reply pass.cpp]: Sure will do :+1:

File: pass.cpp
Comment (akermu) [inline pass.cpp]: Can you put the initialization into the `init` function?

File: README.md
Comment (akermu) [inline README.md]: Can you add a warning that $PASSWORD_STORE_OTP_IDENTIFIER must be initialized before krunner starts, e.g. in `.xprofile` or similar files?

**Review by akermu — CHANGES_REQUESTED** [review]
After that it will be ready to merge :)

**Comment by msmol** [comment]
Done, thanks for the review

**Review by akermu — APPROVED** [review]

**Comment by akermu** [comment]
Thanks again!

**Pull request merged**
